home/src/main/resources/scripts/dumpRestore.bat

```diff
@@ -0,0 +1,46 @@
+@echo off
+
+set action=%1%
+set models=%2%
+set email=%3%
+set password=%4%
+
+set host=http://example:port
+set purge=true
+set restoration_files_path=.
+set dumped_files_path=%restoration_files_path%
+set app_name=vivo
+
+set session=%TEMP%\%RANDOM%
+mkdir "%session%"
+
+set url=%host%/%app_name%/authenticate
+
+set loginName=%email%
+set loginPassword=%password%
+set loginForm=Log in
+
+:: Log in and get session cookies
+curl --cookie-jar "%session%\cookies.txt" --create-dirs -d "loginName=%loginName%" -d "loginPassword=%loginPassword%" -d "loginForm=%loginForm%" %url%
```

Comment:
I think if you try to use login with + sign it wouldn't work here too.

Reply:
It is still -d here. I haven't tested it on Windows, but maybe we should also use --data-urlencode here?

Reply:
You are right, I forgot to add it here for username and password. I will change it right now.